src/components/ScanQRModal/ScanQRWrapper/index.tsx

```diff
@@ -25,22 +29,14 @@ export const ScanQRWrapper = (props) => {
   }
 
   const onScanFinished = (value) => {
-    props.handleScan(value, closeQrModal)
+    handleScan(value, closeQrModal)
   }
 
   return (
     <>
-      <Img
-        alt="Scan QR"
-        className={classes.qrCodeBtn}
-        height={20}
-        onClick={() => {
-          openQrModal()
-        }}
-        role="button"
-        src={QRIcon}
-        testId="qr-icon"
-      />
+      <button onClick={openQrModal} className={classes.qrCodeBtn} title="Scan QR" data-testid="qr-icon" type="button">
```

Comment:
Please, if you can't do it now, in the next iteration, replace this button using the one from safe-react-components.

Reply:
Done, I did need to overwrite some styles. But it seems to be working fine